src/ka/validation.php

```diff
@@ -109,8 +106,6 @@
     |
     */
 
-    'attributes'           => [
-        //
-    ],
+    'attributes' => [],
```

Comment:
Laravel's core validation.php file has been changed and for some reason I changed it as well. https://github.com/laravel/laravel/blob/master/resources/lang/en/validation.php#L112